Branch-SDK/Branch-SDK/Requests/BranchInstallRequest.m

```diff
@@ -48,7 +49,14 @@ - (void)makeRequest:(BNCServerInterface *)serverInterface key:(NSString *)key ca
     params[BRANCH_REQUEST_KEY_IS_REFERRABLE] = @(preferenceHelper.isReferrable);
     params[BRANCH_REQUEST_KEY_DEBUG] = @(preferenceHelper.isDebug);
     
-    [serverInterface postRequest:params url:[preferenceHelper getAPIURL:BRANCH_REQUEST_ENDPOINT_INSTALL] key:key callback:callback];
+    if ([[BNCStrongMatchHelper strongMatchHelper] shouldDelayInstallRequest]) {
+        dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(500 * NSEC_PER_MSEC)), dispatch_get_main_queue(), ^{
+            [serverInterface postRequest:params url:[preferenceHelper getAPIURL:BRANCH_REQUEST_ENDPOINT_INSTALL] key:key callback:callback];
```

Comment:
this is a lighter weight request than the v1/open or v1/install request. so if it is kicked off first it will almost always finish first. I don't want to delay initializing the session longer than that